config/config.exs

```diff
@@ -0,0 +1,6 @@
+use Mix.Config
+
+config :ex_geocode,
+  api_host: "https://maps.googleapis.com/maps/api/geocode"
```

Comment:
I had to edit the `api_host` to remove the path in order to get this to work, ie. `api_host: https://maps.googleapis.com`